docs/source/quickstart.mdx

```diff
@@ -37,16 +39,16 @@ tar -xf path/to/file.tar.gz
 
 ### Running the binary
 
-You can now try running the Apollo Router from your project's root directory with the following command:
+You can now run the Apollo Router from your project's root directory with the following command:
 
 ```bash
-./dist/router
+./router
```

Comment:
I don't think this edit is correct. If you download the release (gzipped tar file) and follow the instructions, you will have a sub-directory `dist` which contains the router, etc...
So, you can either modify the instructions to advise `cd dist` as the first step or re-instate the original version which takes account that the router is in a `dist` sub-directory.

(NB: This comment applies to subsequent edits of this form)

One more option, for sake of completeness, would be to run
```
tar xfz router-0.10.0-x86_64-macos.tar.gz --strip-components=1
```
That would strip the first element of the path and drop the contents into wherever the second path element would be (in this case there isn't one - so the current directory). I don't recommend this, because I'm not sure how widely supported the `--strip-components` option is on different platforms.

Reply:
I think `--strip-components` is widely supported and that we should use it.

Reply:
Thanks for the context y'all! Yeah, I ran the `curl` installer as part of my docs QA and based on the result, incorrectly assumed that the `dist` directory had gone away for all methods across the board.